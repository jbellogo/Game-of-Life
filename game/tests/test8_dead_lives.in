new 8

init
1 0

4 0
4 1

7 0
7 1
7 2

1 2
1 3
1 4
0 2

2 5

3 3
3 4

4 5
5 5
5 4
5 3

7 3
7 4
7 5
6 3

-1 -1
print 
step
print
step
print
steps 6
print
steps 30
print
