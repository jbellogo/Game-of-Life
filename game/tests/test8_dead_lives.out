_X__X__X
____X__X
XX_____X
_X_X_XXX
_X_X_X_X
__X_XX_X
________
________
________
XX____XX
XXX_XX_X
_X___X_X
_X_X___X
__XXXX__
________
________
________
X_X__XXX
__X_XX_X
___X_X_X
_X_X_X__
__XXX___
___XX___
________
__XX____
_X__X___
_XXX____
__X_X___
__XXX___
___X__X_
____XX__
________
________
___XX___
___XX___
________
________
________
________
________
